### PR TITLE
Script to update a-params and fee as per SIP 10

### PR DESCRIPTION
**Pull request by mjdatadog**
called rampA with a final time of 17 days for all a-parameter
changes. Also called setAdminFee() for USDCV2 pool to reduce
admin fee to .03%

### CODE REVIEW

**Review by hammeiam — APPROVED** [review]
Lgtm, I'll run this and confirm the receipt with @penandlim later today before posting on chain

File: scripts/2022_01_19_1_sip_10_update_pool_a_params.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        contract = multisig.contract(pool_addr)
+        contract.rampA(A, now_plus_17_days_seconds)
+        if pool_addr == "0xaCb83E0633d6605c5001e2Ab59EF3C745547C8C7":
+            contract.setAdminFee(3000000000)
+
+    # combine history into multisend txn
+    safe_tx = multisig.multisend_from_receipts()
+
+    # sign with private key
+    safe_tx.safe_nonce = 20
```
Comment (penandlim) [inline scripts/2022_01_19_1_sip_10_update_pool_a_params.py:33]: Remove this line. nonce 20 is already used.

**Review by penandlim — CHANGES_REQUESTED** [review]

File: scripts/2022_01_19_1_sip_10_update_pool_a_params.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from scripts.utils import confirm_posting_transaction
+
+def main():
+    """Ramps A param for select pools as per SIP-10"""
+    print(f"You are using the '{network.show_active()}' network")
+    deployer = accounts.load("deployer")
+    multisig = ApeSafe(MULTISIG_ADDRESSES[CHAIN_IDS["MAINNET"]])
+
+    pools_to_future_a = {
+        "0xa6018520EAACC06C30fF2e1B3ee2c7c22e64196a": 1000, # SaddleALETHPool
```
Comment (hammeiam) [inline scripts/2022_01_19_1_sip_10_update_pool_a_params.py:15]: Our contract code doesn't allow us to move to an arbitrary A value, we can only double the current value at most in one update. Thus these values should be
aleth: 480
btcv2: 400
usdv2: 400

then we'll double them again in a couple weeks.

Comment (mjdatadog) [reply scripts/2022_01_19_1_sip_10_update_pool_a_params.py:15]: btcv2 and usdcv2 are currently both at 400, so will be changing them both to 800

Comment (hammeiam) [reply scripts/2022_01_19_1_sip_10_update_pool_a_params.py:15]: please rm old file

**Pull request merged**